ebs/error_budget.py

```diff
@@ -1,4 +1,4 @@
-"""<Flux-Ratio Error Allocation Kit (FREAK)> module
```

Comment:
sort of an aside but do we want to reconsider the name? Is error budget software a little vague for what we are doing?

Reply:
That should have been deleted.  The "Flux-Ratio ... " name is obsolete.